war/src/main/webapp/css/style.css

```diff
@@ -1975,3 +1975,23 @@ body.main-panel-only #main-panel {
         padding: 2em 10%;
     }
 }
+
+img.icon-sm {
+    width: 16px;
+    height: 16px;
+}
+
+img.icon-md {
+    width: 24px;
+    height: 24px;
+}
+
+img.icon-lg {
+    width: 32px;
+    height: 32px;
+}
+
+img.icon-xl {
```

Comment:
Prob better not to spec the element name either (`img`) as this could easily be applied to a `span` etc if applying the icon as a background image via css. Not a biggie for now though.